### PR TITLE
exchanged images "name" and "tuete" by Christine

### LINKED ISSUE #309: wrong picture Name
The picture of name schould be the same as in the prototyp (see file Name.jpg 
(in Serlo ABC_Medienproduktion/Fehlende Fotos) - the wrong picture is (Name).jpg in the same folger.

### LINKED ISSUE #328: Tüte: better picture
Das Bild zu Tüte ist eine Tasche sollte durch eine Plastiktüte ersetzt werden ( (s. Datei Tüte.jpg 
in Serlo ABC_Medienproduktion/Fehlende Fotos).

### LINKED ISSUE #339: Lobvideo with wrong text
Beim Lobvideo (Mann mit Baby) kommt ein Text mit Tippfehler (Bravo! Du hasts geschafft! Ciao!
Korrektur: Bravo! Du hast es geschafft! Ciao!

### LINKED ISSUE #343: Cafe sounds missing
The PR #308 didn't include the sound files for cafe.

ping @Mariiiam can you add them?

### PR DESCRIPTION
**Pull request by Mariiiam**
fixes #309, fixes #328, fixes #339 and fixes #343 by Christine

### CODE REVIEW

**Comment by Knorrke** [comment]
@Mariiiam please don't close the issues before merging ;-) (also: since you use "fixes" in your Pull-Request description they get closed automatically)

File: src/assets/words/sounds.ts [hunk 1]
```diff
@@ -54,6 +54,8 @@ const sounds: { [id: string]: () => AssetTypes.SoundAsset } = {
   brot_short: () => require('./sounds/brot_short.mp3'),
   buch_short: () => require('./sounds/buch_short.mp3'),
   bus_short: () => require('./sounds/bus_short.mp3'),
+  cafe_short: () => require('./sounds/cafe_short.mp3'),
```
Comment (inyono) [inline src/assets/words/sounds.ts]: Please add the sound files to `packages/assets/words/sounds`

**Comment by inyono** [comment]
Also: please add the images to `packages/assets/words/images` instead. The images in `src/assets/words/images` get generated from that (and resized / optimized if needed). Same for sounds

**Comment by Knorrke** [comment]
script for images wasn't working on christines laptop, I'll try that later then it's good to review @inyono

**Pull request merged**